techniques/system/common/1.0/update.st

```diff
@@ -387,11 +387,13 @@ bundle agent update_action
 bundle agent check_uuid
 {
   classes:
-    # Abort if no uuid is defined
-    "should_not_continue" expression => "wrong_uuid|no_uuid";
-    "no_uuid" not => fileexists("${g.uuid_file}");
+      # Abort if no uuid is defined
+      "should_not_continue" expression => "wrong_uuid|no_uuid";
+    community_edition::
+      "no_uuid" not => fileexists("${g.uuid_file}");
 &if(!INITIAL)&
-    "wrong_uuid" not => strcmp("&UUID&", "${g.uuid}");
+    any::
+      "wrong_uuid" not => strcmp("&UUID&", "${g.uuid}");
```

Comment:
how can `g.uuid` be defined if `uuid_file` does not exist?

Reply:
yes but doesn't g.uuid come from uuid_file?

Reply:
yes

Reply:
ha, you want to check it only if file is there ?